src/integTest/groovy/com/bmuschko/gradle/docker/ToolingApiIntegrationTest.groovy

```diff
@@ -60,4 +60,8 @@ repositories {
     protected String createUniqueImageId() {
         "gradle/${UUID.randomUUID().toString().replaceAll('-', '')}"
     }
+
+    protected String createUniqueContainerName() {
```

Comment:
Finally I realized that a container name may not contain a '/' like the image id. So I have kept the 2 methods.